tests/scripts/set-release-ver.sh

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+FROM_TAG=$(grep "git clone --single-branch" Documentation/Getting-Started/quickstart.md | awk '{ print $6 }')
+TO_TAG="$1"
+
+# This script is supposed to facilitate the preparation of a rook release by
+# changing example and documentation files to use the new release version instead the old one.
+#
+# It will usually be run on a release branch and the resulting changes will be the content of the
+# commit for the basis of the release.
+
+# The script needs two version tags:
```

Comment:
Only one version tag now

Reply:
iu ses two but only one needs to be passed as argument. Wording updated.